doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -142,6 +142,8 @@ Matter
   You can use this class to generate the Spake2+ verifier at runtime.
   To use this class, enable the :kconfig:option:`CONFIG_PSA_WANT_ALG_PBKDF2_HMAC` and :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_SPAKE2P_KEY_PAIR_DERIVE` Kconfig options.
 
+* Enabled the ZMS file subsystem in all devices that contain MRAM, such as the nRF54H Series devices.
```

Comment:
```suggestion
* Updated:

  * Whatever there is now under Matter samples
  * All Matter samples to enable the ZMS file subsystem in all devices that contain MRAM, such as the nRF54H Series devices.
```
Now, this complies with our changelog wording conventions. Please move this under Matter samples (Updated).